src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java

```diff
@@ -92,19 +94,32 @@ public void savesAMember() throws Exception {
     }
 
     @Test
-    public void updatesAMember() throws Exception {
-        aMember.setUsername("aNewUsername");
-        mockMvc.perform(put("/members/1")
+    public void updatesAnExistingMember() throws Exception {
+        aMember.setEmail("newExample@email.com");
+        mockMvc.perform(put(String.format("/members/%d", aMember.getId()))
             .contentType(MediaType.APPLICATION_JSON)
             .accept(MediaType.APPLICATION_JSON)
             .content(mapper.writeValueAsString(aMember)))
             .andExpect(status().isOk())
-            .andExpect(jsonPath("$.username", is(aMember.getUsername())));
+            .andExpect(jsonPath("$.email", is(aMember.getEmail())));
+    }
+
+    @Test
+    public void createsANewMemberWhenIdNotFound() throws Exception {
+        Member anotherMember = new Member();
+        anotherMember.setUsername("anotherMemberUsername");
+
+        mockMvc.perform(put(String.format("/members/%d", aMember.getId()+1))
+            .contentType(MediaType.APPLICATION_JSON)
+            .accept(MediaType.APPLICATION_JSON)
+            .content(mapper.writeValueAsString(anotherMember)))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.username", is(anotherMember.getUsername())));
     }
 
     @Test
     public void deletesAMember() throws Exception {
-        mockMvc.perform(get(String.format("/members/%d", aMember.getId())))
+        mockMvc.perform(delete(String.format("/members/%d", aMember.getId())))
```

Comment:
Can't believe I missed this last time. My bad

Reply:
Yeah, I kicked myself when I saw it